CONTRIBUTING.md

```diff
@@ -1,3 +1,40 @@
 # Contributing
 
 TODO
+
+### Adding a new icon
+
+1. Add SVGs to Size-Specific Folders
+- Get the three versions (16px, 24px, 32px) of the SVG icon you want to add.
+- Place each version in the following folders:
+  - `packages/svg-icons/src/icons/16px`
+  - `packages/svg-icons/src/icons/24px`
+  - `packages/svg-icons/src/icons/32px`
+
+2.Optimization and Generation of icons
```

Comment:
There's a space missing between. And Optimization only mentioning as it can mess markdown prices of numeric list.